index.js

```diff
@@ -34,9 +33,22 @@ const _analyzeLine = (alineAttrs, apool) => {
 exports.getLineHTMLForExport = async (hookName, context) => {
   const align = _analyzeLine(context.attribLine, context.apool);
   if (align) {
-    context.lineContent =
-      `<p style='text-align:${align}'>${Security.escapeHTML(context.text.substring(1))}</p>`;
-    return `<p style='text-align:${align}'>${Security.escapeHTML(context.text.substring(1))}</p>`;
+    if (context.text.indexOf('*') === 0) {
+      context.lineContent = context.lineContent.replace('*', '');
+    }
+    const heading = context.lineContent.match(/<h([1-6])([^>]+)?>/);
+
+    if (heading) {
+      if (heading.indexOf('style=') === -1) {
+        context.lineContent = context.lineContent.replace('>', ` style='text-align:${align}'>`);
+      } else {
+        context.lineContent = context.lineContent.replace('style=', `style='text-align:${align} `);
```

Comment:
The content isn't being escape here any more dude???  Isn't that a problem? 👍

Reply:
If you check `ep_headings2` plugin it isn't escaped there too as it will remove any html added by other plugins. Probably one way to fix would be to check line with `indexOf('script') > -1'` and if does then use `Security.escapeHTML`. Would that be acceptable?

Reply:
hrm, I'm sure at some point we addressed this..   script wouldn't suffice, too easy to manipulate...

Reply:
Well I don't see other plugins using `Security.escapeHTML` during htmlExport: